tachyon/zk/plonk/constraint_system/constraint_system.h

```diff
@@ -818,6 +824,7 @@ class ConstraintSystem {
   size_t num_fixed_columns_ = 0;
   size_t num_advice_columns_ = 0;
   size_t num_instance_columns_ = 0;
+  size_t num_simple_selectors_ = 0;
   size_t num_selectors_ = 0;
```

Comment:
This leads us to make additional code changes to be compatible with Halo2, but this will be more consistent with num_simple_selectors_. I'll add this change on top.